.pre-commit-config.yaml

```diff
@@ -1,12 +1,12 @@
 repos:
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: "v4.0.0-alpha.8"
+    rev: "v3.1.0"
     hooks:
       - id: prettier
         additional_dependencies:
-          - prettier@3.5.0
+          - prettier@3.6.2
   - repo: https://github.com/python-jsonschema/check-jsonschema
-    rev: 0.33.1
+    rev: 0.29.4
```

Comment:
keep the latest versions of the other ones please. I will then try to let renovate keep them up to date

Reply:
```suggestion
    rev: 0.33.1
```